src/InteractiveWindow/Version.props

```diff
@@ -1,5 +1,5 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RoslynSemanticVersion>1.2.0</RoslynSemanticVersion>
+    <RoslynSemanticVersion>2.0.0</RoslynSemanticVersion>
```

Comment:
We should probably just remove this props file.  It was only there because we originally weren't generating binding redirects for it.